app/kumactl/cmd/root.go

```diff
@@ -111,7 +111,7 @@ func NewRootCmd(root *kumactl_cmd.RootContext) *cobra.Command {
 	cmd.AddCommand(inspect.NewInspectCmd(root))
 	cmd.AddCommand(install.NewInstallCmd(root))
 	cmd.AddCommand(uninstall.NewUninstallCmd())
-	cmd.AddCommand(version.NewVersionCmd())
+	cmd.AddCommand(version.NewCmd(root))
```

Comment:
So we gonna have 2 `version` command: `github.com/kumahq/kuma/app/kumactl/cmd/version` and `github.com/kumahq/kuma/pkg/cmd/version`? Maybe we can create method `BuildInfo.Print`, to have this formatting in one place:
```go
cmd.Println(fmt.Sprintf("Product:    %s", kuma_version.Product))
cmd.Println(fmt.Sprintf("Version:    %s", buildInfo.Version))
cmd.Println(fmt.Sprintf("Git Tag:    %s", buildInfo.GitTag))
cmd.Println(fmt.Sprintf("Git Commit: %s", buildInfo.GitCommit))
cmd.Println(fmt.Sprintf("Build Date: %s", buildInfo.BuildDate))
```

Reply:
done